### DISCUSSION TITLE
Support for learning goal and/or certification creation for MJML

### DISCUSSION THREAD

**Discussion by Weissheiten** [mjmlio/mjml#2590] [Ideas]
The MJML guide, documentation and articles/tutorials section are a great starting point for MJML. For educational/training purposes clear hard skill definitions and general defined learning goals are missing however.

Once created - these skill definitions help the community using MJML to quickly tag their learning resources with the appropriate skills and allows everyone learning MJML to get a clear overview of their progress and identify the next suitable learning resources.
(Scenarios include monitoring the own learning progress and getting suggestions for the next suitable learning resource directly on the MJML website, claiming badges for reaching milestones, etc.)

We'd like to support you in creating such definitions with SkillDisplay, as having those would not only support the MJML community, but also be beneficial to educational partners (schools, universities, training centers) and allow us in turn to broaden the available skill portfolio on the platform. We have six years of experience - including the lead of a 3-year Erasmus+ EU project in the field and work with certification authorities on the open market. 

Feel free to contact me to schedule a hangout to have a chat about the scenario and thank you for creating MJML.

**Comment by iRyusa** [comment]
Not interested, thanks.
